nms/app/packages/magmalte/app/components/feg/FEGGatewayDialog.js

```diff
@@ -125,10 +123,27 @@ function getDiameterServerConfig(
   };
 }
 
-function getOverwriteAPN(overwrite_apn: ?string): overWriteAPN {
-  return {
-    apn: overwrite_apn || '',
-  };
+function getVirtualApnRules(rules: ?virtual_apn_rules): 
+  ?Array<[string, string]> {
+  if (!rules || rules.length == 0) {
```

Comment:
minor nit: this might be unnecessary. We probably would want getVirtualApnRules to return empty list when rules don't exist.